SwinjectAutoregistration.xcodeproj/project.pbxproj

```diff
@@ -833,6 +833,7 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = CD5B9E301D832A4F007ED05F /* iOS-Framework.xcconfig */;
 			buildSettings = {
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = arm64;
```

Comment:
This is not needed and not wanted. If you strip this arch this repo cannot be used in M1 simulators!!